src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,60 @@
 // Vendor-specific styles.
+
+// UIO
+
+// uio tab
+#defaultContainer .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
+	background: $white;
+	border: rem(1) solid $green-dark;
+	border-radius: 0 0 rem(30) rem(30);
+	border-top: none;
+	box-shadow: none;
+	box-sizing: border-box;
+	padding: rem(10) rem(20);
+
+	button {
+		color: $black;
+		font-family: "Roboto", sans-serif;
+		font-size: rem(16);
+		font-style: normal;
+		font-weight: $font-weight-normal;
+		line-height: rem(19);
+	}
+
+	&:hover,
+	&:focus-within {
+		border: rem(3) solid $green-dark;
+		border-top: none;
+		margin-right: rem(70);
+
+		button {
+			font-weight: $font-weight-bold;
+		}
+	}
+}
+
+// focus
+a[href="https://www.ic.gc.ca/eic/site/118.nsf/eng/home"],
```

Comment:
Having css depend on a href value may not be a good idea as this value is a part of content rather than common css selectors. Please consider to use a selector such as `footer .funders a`.

The same comment applies to line 38, 39.

Reply:
`footer .funders a` works for now but once we add more funders we will need another way to increase specificity